pyartcd/pyartcd/pipelines/build_layered_products.py

```
@@ -222,6 +223,26 @@ async def _rebase(self, image_list: str) -> str:
+        built_images = [entry['name'] for entry in records if not int(entry['status'])]
+        failed_images = [entry['name'] for entry in records if int(entry['status'])]
+
+        desc_parts = [f'{len(built_images)} image(s) succeeded']
+        if failed_images:
+            desc_parts.append(f'{len(failed_images)} image(s) failed')
+        jenkins.update_description(f'{", ".join(desc_parts)}<br/>')
+
+        if 1 <= len(failed_images) <= 10:
+            jenkins.update_description(f'Failed images: {", ".join(failed_images)}<br/>')
+        elif len(failed_images) > 10:
+            jenkins.update_description(f'Check record.log for the full list of failed images<br/>')
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)